tests/unit/CliftType.cpp

```diff
@@ -0,0 +1,48 @@
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "mlir/IR/Diagnostics.h"
+
+#include "revng-c/mlir/Dialect/Clift/IR/Clift.h"
+#include "revng-c/mlir/Dialect/Clift/Utils/ImportModel.h"
+
+// Don't reorder this
+#include "revng/tests/unit/ModelType.inc"
```

Comment:
Usually this ends up at the end of the file. I guess you're forward declaring the static functions in there.
This can also work.

Reply:
I moved it to the end of the file. That works too.